src/helpers/SettingsHelper.js

```diff
@@ -40,8 +40,18 @@ const SettingsHelper = ({
       ...fields,
       alert: '',
     };
-    parameter[event.target.id] = event.target.value;
-    setFields(parameter);
+    if (event.target.files && event.target.files[0]) {
```

Comment:
saw it 2nd time, move this to some helper

Reply:
Solved